### ISSUE TITLE
Page does not Load

### ISSUE THREAD

**Issue by axleross09** [cypress-io/cypress#7357]
### Current behavior:
Page does not load correctly, Tried using cy.visit() and manually navigating in the Cypress Browser and the page is not loading. It is also throwing an error `Error: EISDIR: illegal operation on a directory`.

Screen Recording:
https://cbo.d.pr/vTil2B

### Desired behavior:

Page will Load

### Test code to reproduce

current test script block:
```js
it('Add Valid Company Details with Required Fields only', () => {
  cy.login()
  cy.wait(5000)
  cy.visit('/admin/insites/#/crm/companies')
  //Assert that table will load
  //Get Add Company Button
  //Assert URL
  /*
  Notes to dev:
  Add Test Attributes on all Tabs
  Add Test Attributes to all required fields and elements
  Add Test Attributes to all Buttons
  */
  //Get required fields and type valid data
  //Get Save button (Save only) and Click
  //Assert if Company Added is displayed on List
})
```

I have also created a custom config file 

```json
{
    "baseUrl": "baseURL",
    "chromeWebSecurity": false,
    "defaultCommandTimeout": 50000,
    "pageLoadTimeout": 50000,
}
```

**Comment by axleross09** [comment]
Upon checking again, It seems that another Cypress Browser opens inside the cypress browser when opening a different page.

![Screenshot 2020-05-15 at 2 21 53 PM](https://user-images.githubusercontent.com/65379921/82018268-88693800-96b7-11ea-986e-7f39885aea2f.png)

**Comment by jennifer-shehane** [comment]
Duplicate of #3975 

A fix for this will be released in our next version, requiring an experimental flag to be passed. Please follow the above issue for updates.

**Comment by jennifer-shehane** [comment]
This fix is available starting in [`4.6.0`](https://docs-staging.cypress.io/guides/references/changelog.html#4-6-0) as an [experiment](https://on.cypress.io/experiments) which you can access by setting this config option in your `cypress.json` or elsewhere:

```json
{
	"experimentalSourceRewriting": true
}
```

The fix is experimental, so there may be some situations where the this is not fixed. 

If you're still this issue while setting the `experimentalSourceRewriting` to true in 4.6.0 - [open a new issue](https://github.com/cypress-io/cypress/issues/new) with a reproducible example + screenshots, etc - filling out our issue template.
